homeassistant/components/template/strings.json

```
@@ -57,6 +57,19 @@
         },
         "title": "Template image"
       },
+      "event": {
+        "data": {
+          "device_id": "[%key:common::config_flow::data::device%]",
+          "name": "[%key:common::config_flow::data::name%]",
+          "device_class": "[%key:component::template::config::step::sensor::data::device_class%]",
+          "event_type": "Last fired event type",
+          "event_types": "Event types"
```

This could be referenced - similar to other template types further down in strings.json:

```suggestion
          "event_types": "[%key:component::event::entity_component::_::state_attributes::event_types::name%]"
```